howtos/setup.md

```diff
@@ -14,6 +14,12 @@ npm run start
 
 This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
 
+Simplify the development process by installing the pre-commit hook once, which will automatically lint and format your staged files before each commit.
+
+```bash
+npm run prepare
+```
```

Comment:
I'm under the impression that running `npm install` (as specified above, under "Installation") [will run the prepare task](https://docs.npmjs.com/cli/v10/using-npm/scripts#npm-install). Was that not happening for you?

Reply:
Interesting, based on the docs I'd agree that it should run on install. Somehow it never triggered at me, so I'd assume some local setup issues or an issue in npm at the time of installation. Read some posts that could indicate that it can happen.
in this case I'd close the PR as it seems to be an isolated issue.